downstream/modules/platform/con-backup-aap.adoc

```diff
@@ -0,0 +1,16 @@
+[id="con-backup-aap_{context}"]
+
+= Back up your {PlatformNameShort} instance
+
+Back up an existing {PlatformNameShort} instance by running the `.setup.sh` script with the `backup_dir` flag, which saves the content and configuration of your current environment:
+
+. Navigate to your `ansible-tower-setup-latest` directory.
+. Run the `./setup.sh` script following the example below:
++
+----
+$ ./setup.sh -e ‘backup_dir=/ansible/mybackup’ -e ‘use_compression=True’ @credentials.yml -b <1><2>
+----
+<1> `backup_dir` specifies a directory to save your backup to.
+<2> `@credentials.yml` passes the password variables and their values encrypted via `ansible-vault`.
+
+With a successful backup, a backup file will be created at `/ansible/mybackup/tower-backup-latest.tar.gz` .
```

Comment:
Suggest active voice: Change "a backup file will be..." to "a backup file is..."